packages/react-positioning/src/utils/getPopperOffset.test.ts

```diff
@@ -0,0 +1,66 @@
+import { OffsetFunction } from '../types';
+import { PopperOffsetFunction, getPopperOffset } from './getPopperOffset';
+
+describe('getPopperOffset', () => {
+  it('should ignore object offsets', () => {
+    const offset = [10, 10];
+    const transformedOffset = getPopperOffset({ crossAxis: 10, mainAxis: 10 });
+    expect(transformedOffset).toEqual(offset);
+  });
+
+  it('should ignore number offsets', () => {
+    const offset = 10;
+    const transformedOffset = getPopperOffset(offset);
+    expect(transformedOffset).toEqual([0, offset]);
+  });
+
+  it('should keep function offset as function', () => {
+    const dummyRect = { x: 0, y: 0, width: 0, height: 0 };
```

Comment:
```suggestion
    const dummyRect: Rect = { x: 0, y: 0, width: 0, height: 0 };
```

nit: It would be nice to use types there